searchlib/src/vespa/searchlib/attribute/attribute_header.h

```diff
@@ -63,6 +63,8 @@ class AttributeHeader {
     bool getEnumerated() const { return _enumerated; }
     uint64_t getCreateSerialNum() const { return _createSerialNum; }
     uint32_t getVersion() const  { return _version; }
+    uint64_t get_total_value_count() const { return _totalValueCount; }
```

Comment:
Please test these two in verify_roundtrip_serialization() in attribute_header_test.cpp.